### ISSUE TITLE
Stable channel support for Gitlab

### ISSUE THREAD

**Issue by uilianries** [conan-io/conan-package-tools#50]
Hi!

At [packager.py](https://github.com/conan-io/conan-package-tools/blob/develop/conan/packager.py):470, we have:
```python
def _get_channel(self, default_channel, stable_channel)
```
This function filter the CI runner by environment variable, but Gitlab is not listed.  As result, **CONAN_STABLE_BRANCH_PATTERN** doesn't work at Gitlab Runner.

To support this new feature, you could use [Gitlab Variables](https://docs.gitlab.com/ce/ci/variables/).

I think that, this piece solves the case:
```python
gitlab = os.getenv("GITLAB_CI" , False)  # Mark that job is executed in GitLab CI environment
gitlab_branch = os.getenv("CI_BUILD_REF_NAME", False)  # The branch or tag name for which project is built
```

Saludos.

**Comment by lasote** [comment]
Hi! I've released conan_package_tools==0.2.36 with support for those variables.
Let me know if it works well!

Salud!

**Comment by uilianries** [comment]
I validated now and this new release worked very well on Gitlab runner.

Gracias!
